check_nexenta.py

```diff
@@ -506,15 +521,16 @@ def collect_perfdata(nexenta):
                 perfdata.append("'/%s compressratio'=%s" % (vol, ratio[:-1]))
 
         # Get memory used, free and paging.
-        memstats = api.get_data(obj='appliance', meth='get_memstat', par=[''])
```

Comment:
`par=['']` fails with:

`Fewer items found in D-Bus signature than in Python arguments`

Passing a real empty array fixes this issue.